services/ble/Gap.cpp

```diff
@@ -106,4 +106,86 @@ namespace services
     {
         GapCentralObserver::Subject().StopDeviceDiscovery();
     }
+
+    infra::ConstByteRange GapAdvertisingDataParser::LocalName(infra::ConstByteRange data)
+    {
+        auto localName = ParserAdvertisingData(data, GapAdvertisementDataType::shortenedLocalName);

```

Comment:
Swap shortened & full (cococo (and we) only use full, so saves an iteration

Reply:
Done.